### PR TITLE
Correct ITs for NonEmptyAtclauseDescription and JavadocTagContinuationIndentation

### PR DESCRIPTION
**Pull request by Vladlis**
In NonEmptyAtclauseDescriptionTest the only way I found possible to write "//warn" comments in the input file is
```java
/**
 	 * Some javadoc.  
 	 * @param c     	 */ //warn
```
But this would make the test incomplete, I guess.

### CODE REVIEW

**Comment by romani** [comment]
Please ignore failure of codecov, we experimenting with it

File: src/it/java/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/JavadocTagContinuationIndentationTest.java [hunk 1]
```diff
@@ -2,58 +2,38 @@
...
-        
-        Integer[] warnList = builder.getLinesWithWarn(filePath);
-        verify(checkConfig, filePath, expected, warnList);
-    }
-    
     @Test
-    public void noFinalizerExtendedTest() throws Exception {
-        
-        String msg = getCheckMessage(NoFinalizerCheck.class, "avoid.finalizer.method");
+    public void testDefault() throws Exception {
```
Comment (mkordas) [inline src/it/java/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/JavadocTagContinuationIndentationTest.java]: > `testDefault()`

I would make it more descriptive like `testWithDefaultConfiguration`

File: src/it/java/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/NonEmptyAtclauseDescriptionTest.java [hunk 2]
```diff
@@ -8,54 +10,45 @@
...
+        String msg = getCheckMessage(NonEmptyAtclauseDescriptionCheck.class, "non.empty.atclause");
 
         final String[] expected = {
-            "9: " + msg,
-            "21: " + msg,
-            "33: " + msg,
-            "45: " + msg,
-            "57: " + msg,
-            "69: " + msg,
+            //this is a case with description that is sequences of spaces
```
Comment (mkordas) [inline src/it/java/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/NonEmptyAtclauseDescriptionTest.java]: 1) I think usually we put space after `//`
2) `sequences` -> `sequence`
3) We have multiple same comments here... My proposition is to create two input files, but one with spaces only (lines 27-39) and second with others (40-94). If they have proper test names, comments will become redundant.

File: src/it/java/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/NonEmptyAtclauseDescriptionTest.java [hunk 3]
```diff
@@ -8,54 +10,45 @@
...
+            "94: " + msg,
         };
 
-        Configuration checkConfig = builder.getCheckConfig("NoFinalizer");
-        String filePath = builder.getFilePath("NoFinalizeExtendInput");
+        Configuration checkConfig = builder.getCheckConfig("NonEmptyAtclauseDescription");
+        String filePath = builder.getFilePath("InputNonEmptyAtclauseDescriptionCheck");
         
-        Integer[] warnList = builder.getLinesWithWarn(filePath);
+        Integer[] warnList = {27, 28, 29, 38, 39, 40, 76, 77, 78, 79, 80, 81, 90, 91, 92, 93, 94};
```
Comment (mkordas) [inline src/it/java/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/NonEmptyAtclauseDescriptionTest.java]: > `{27, 28, 29, 38, 39, 40, 76, 77, 78, 79, 80, 81, 90, 91, 92, 93, 94}`

Can we do something about it? Looks like duplication - any change in input file would force us to update line numbers in two places.

**Comment by mkordas** [comment]
@Vladlis 

My review is done. Please fix the issues and then @romani can merge. Thanks for fixing that, these tests were ridiculous!

**Comment by romani** [comment]
@Vladlis, please fix issues from mkordas and let me validate your code.

**Comment by Vladlis** [comment]
> I would make it more descriptive like ```testWithDefaultConfiguration```

fixed
> My proposition is to create two input files

done
> Looks like duplication

fixed

**Comment by romani** [comment]
merged as FF

**Pull request closed without merging**